services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -116,8 +197,36 @@ class HankeController(
         return createdHanke
     }
 
-    /** Update one hanke. */
-    @PutMapping("/{hankeTunnus}")
+    @PutMapping(
+        "/{hankeTunnus}",
+        consumes = [APPLICATION_JSON_VALUE],
+        produces = [APPLICATION_JSON_VALUE]
+    )
+    @Operation(
+        summary = "Update hanke",
+        description =
+            "Update an existing hanke. An updated hanke must comply with the restrictions defined in HankeValidator."
```

Comment:
This could also mention the uatomatic status update, and that the tormaystarkastelu is run each time the hanke is updated.

Reply:
Description updated.